### DISCUSSION TITLE
Maybe Svelte?

### DISCUSSION THREAD

**Discussion by SrGeneroso** [firecmsco/firecms#290] [Ideas]
I just discovered this marvelous solution. I was eager to find an opensource backend for firebase that doesn't rely in functions (at least as far as I know).
I'm doing some work with sveltekit and I'm waiting into final release to deploy some apps, until then, I'll play around with this backend and see if it could be done with sveltekit as well. Anyone interested?

**Comment by bn-l** [comment]
Same here. @fgatti675 awesome project. When I looked at the demo I was already in the "shut up and take my money" mood. The lack of svelte is a deal breaker for me though. I won't go back to react.

**Comment by fgatti675** [comment]
Hi @SrGeneroso and @bn-l 
So the complete app is built using React and of course it will have to stay that way.
I understand you like Svelte, but the only alternative I can suggest is trying to use a wrapper that allows to use Svelte inside React?
https://github.com/Rich-Harris/react-svelte
It is a bit of a hack but I guess it could work. Let me know how it goes if you try it out ;)
